.travis.yml

```diff
@@ -44,6 +44,8 @@ install:
   -o gae-download.zip
 - unzip -qq gae-download.zip -d $TOOLS_DIR/google_appengine_1.9.19/
 - rm gae-download.zip
+- bash scripts/install_third_party.sh
+- python scripts/pre_commit_linter.py --path=. --silent
```

Comment:
As per conversation, will be separated into a separate test like the frontend, backend, and coverage tests.